core/vm-common/executionHandler.go

```diff
@@ -0,0 +1,13 @@
+package vmcommon
+
+// VMExecutionHandler interface for any Elrond VM endpoint
+type VMExecutionHandler interface {
```

Comment:
please move in the next file (interface.go)

Reply:
done